src/coreclr/vm/excep.cpp

```diff
@@ -224,9 +224,11 @@ void GetExceptionMessage(OBJECTREF throwable, SString &result)
 
     STRINGREF pString = GetExceptionMessage(throwable);
 
+    GCPROTECT_BEGIN(pString);
```

Comment:
This should not be needed. `GetSString` does not trigger GC, so it is not necessary to GC protect the object reference around it.